Sources/SwiftDocC/Model/TaskGroup.swift

```diff
@@ -238,3 +240,8 @@ extension TaskGroup {
     }
 }
 
+private extension SourceRange {
+    var source: URL? {
+        lowerBound.source ?? upperBound.source
```

Comment:
Out of curiosity, is there a case where lowerBound.source may not exist but upperBound.source would exist?

Reply:
In theory it's possible but I don't think there's a real case where it would happen.. I believe in practice either one or both of the source locations would have a source.